es-core/src/AudioManager.cpp

```diff
@@ -3,7 +3,7 @@
 #include <SDL.h>
 #include "Log.h"
 
-std::vector<std::shared_ptr<Sound>> AudioManager::sSoundVector;
+std::vector<std::shared_ptr<Sound> > AudioManager::sSoundVector;
```

Comment:
I guess it depends on the C++ version you're aiming for.
https://stackoverflow.com/questions/7087033/for-nested-templates-when-did-become-standard-c-instead-of